lib/chatBot.js

```diff
@@ -795,7 +828,7 @@ ChatBot.prototype._onChatStateChange = function(stateChange, chatterActedOn, ste
 	}
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unexpected use of '&'.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/944346627/issues/source?bid=1980036&fileBranchId=2848073#l830)